Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -135,6 +135,26 @@ public final class ProjectPamphletViewController: UIViewController {
         self?.goToRewards(project: project, refTag: refTag)
       }
 
+    self.viewModel.outputs.goToManageViewPledge
```

Comment:
I'm seeing a lot of inconsistencies between the naming of VMs `output` signal and the `private` action on the controller...do you think it would be worth it cleaning this up..

So for example:

`self.viewModel.outputs.goToManageViewPledge` would trigger `self?.goToManageViewPledge`, etc...

For the `deprecated` ones I'd suggest we simply add it as a suffix:

`self.viewModel.outputs.goToViewBackingDeprecated` links to `self?.goToViewBacking`...

Reply:
It makes sense to change the `private` actions to match the name of the output. Will change that.

Regarding the `deprecated` naming, I'm following the naming convention that we adopted for deprecated classes (`DeprecatedCheckoutViewModel`, `DeprecatedRewardPledgeViewController`, `DeprecatedRewardCell` etc)